examples/datasource_mapping/resources.yaml

```diff
@@ -0,0 +1,241 @@
+---
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: grafana
+  labels:
+    dashboards: "grafana"
+spec:
+  client:
+    preferIngress: true
```

Comment:
The example is not entirely correct given the fact that now operator expects ingress spec. I think, in general case, it's easier not to set `preferIngress` at all.

Reply:
ah, good point. Will remove this.

Reply:
I fixed this in the example.